app.rb

```diff
@@ -0,0 +1,23 @@
+require 'sinatra/base'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+
+  get '/' do
+    @peeps = Peep.all
+    erb(:view_peeps)
+  end
+
+  get '/new' do
+    erb(:add_peep)
+  end
+
```

Comment:
Like that you directly use the params to add info to your .add method rather than save the info into variables and then pass that in!